packages/core/src/common/props.ts

```diff
@@ -80,6 +80,7 @@ export interface IOptionProps extends IProps {
 
 /** A collection of curated prop keys used across our Components which are not valid HTMLElement props. */
 const INVALID_PROPS = [
+    "active",
```

Comment:
I couldn't find any valid HTMLElement props that have `active` but let me know if that's wrong to filter at this stage or if I should filter in `Button`